api/src/services/generate-xform.js

```diff
@@ -18,9 +18,11 @@ const FORM_STYLESHEET = path.join(__dirname, '../xsl/openrosa2html5form.xsl');
 const MODEL_STYLESHEET = path.join(__dirname, '../../node_modules/enketo-xslt/xsl/openrosa2xmlmodel.xsl');
 const XSLTPROC_CMD = 'xsltproc';
 
-const stdinErrorHandler = function (xsltproc, err, reject) {
+const processErrorHandler = (xsltproc, err, reject) => {
   xsltproc.stdin.end();
-  if (err.code === 'EPIPE') {
+  if (err.code === 'EPIPE'                                                    // Node v10,v12,v14
+      || (err.code === 'ENOENT' && err.syscall === `spawn ${XSLTPROC_CMD}`)   // Node v8,v16+
```

Comment:
Interesting that Node v10, v12 and v14 behave the same while Node v16 behaves like it used to be in Node v8
🤔 Node 16 regressing to Node 8 for this scenario?

Reply:
Yes, but I couldn't find any documentation about it, the official docs from Nodejs.org has little information about the errors.